src/PerfView/GcStats.cs

```diff
@@ -1051,7 +1104,21 @@ private static string PrintCondemnedReasonsToHtml(TraceGC gc)
             new string[] {"Compacting<BR/>Full<BR/>GC", "Last GC we trigger before we throw OOM"},
             new string[] {"Small<BR/>Heap", "Heap is too small for doing a background GC and we do a blocking one instead"},
             new string[] {"Ephemeral<BR/>Before<BR/>BGC", "Ephemeral GC before a background GC starts"},
-            new string[] {"Internal<BR/>Tuning", "Internal tuning"}
+            new string[] {"Internal<BR/>Tuning", "Internal tuning"},
+            new string[] { "Almost_max_alloc", "Almost_max_alloc"},
```

Comment:
let's chat about this on Monday to have more meaningful text here. the text should help folks to understand what this reason is for. eg in this [SO issue](https://stackoverflow.com/questions/59130015/condemned-generation-is-not-garbage-collected) the final generation is different from generation and the "Avoid_unproductive" reason should have text that explains this exact issue.